### ISSUE TITLE
Keyword "inertial" for connector variable (ThermoFluidStream)

### ISSUE THREAD

**Issue by MarekMatejak** [modelica/ModelicaSpecification#3467]
The pattern of "inertial pressure" calculated as difference between "pressure" and "steady pressure" from ThermoFluidStream library can be generalized even for chemical domain as a "inertial electro-chemical potential" and may be also for another domains.

The idea is to hide topology (e.g. implemented in ThermofluidStream.Topology.JunctionNM or in Chemical.Topology.JunctionNM https://github.com/MarekMatejak/Chemical/tree/ThermofluidStream ) behind keyword "inertial" in connector variable.  These equations in case of "inertial" connector variable could be generated in similar way that "flow" or "stream" connector variable generates equations.

The goal of this should be to simplify definition of these kind of models for users, because they do not need to explicitly use splitters or junctions.

![image](https://github.com/modelica/ModelicaSpecification/assets/6085966/2739c8e7-3935-4cc0-9b26-1e1de5e81078)

**Comment by HansOlsson** [comment]
Can you explain this in more detail? I don't fully understand this; is it "steady" as a fixed inertial frame of reference (like atmosphere pressure), or a possibly time-varying "steady-state" pressure?

**Comment by MarekMatejak** [comment]
I mean time-varying "steady mass flow pressure" as described by Dirk Zimmer (DLR) in "Robust object-oriented formulation of directed thermofluid stream networks" https://www.tandfonline.com/doi/full/10.1080/13873954.2020.1757726 .

**Comment by HansOlsson** [comment]
I think this will require a lot more than just adding a keyword, but I haven't investigated it in detail.

**Comment by MarekMatejak** [comment]
Yes, this issue is definitely not a trivial thing. However, it could make Modelica much user friendly for these types of models. Should I prepare some initial materials for this issue?
